src/components/app-wrapper/app-wrapper.stories.js

```diff
@@ -6,8 +6,8 @@ import AppWrapper from './app-wrapper';
 
 storiesOf('App Wrapper', module)
   .add('default', () => {
-    const children = text('children', 'This component will wrap itschildren within '
-    + 'the width constraints of your application.');
+    const children = text('children', 'This component will wrap its children within '
+      + 'the width constraints of your application.');
```

Comment:
Can we use template literal here?

Reply:
appWrapper component is not a part of this ticket -I just found a typo. I also know that this part is already refactored by Michał and waiting for merge